api.md

```diff
@@ -779,7 +779,237 @@ Get all chunks older than 3 months and newer than 4 months:
 SELECT show_chunks(older_than => interval '3 months', newer_than => interval '4 months');
 ```
 
+## reorder_chunk() [](reorder_chunk)
+Reorder a chunk's heap to follow the order of an index. This function acts
+similarly to the PostgreSQL CLUSTER command (see:
+https://www.postgresql.org/docs/current/sql-cluster.html), however it uses lower
+lock levels so that the chunk and hypertable are able to be read for most of the
```

Comment:
so that, unlike with the CLUSTER command, the chunk and hypertable ... most of the reorder process.